.github/workflows/docker.yaml

```diff
@@ -0,0 +1,71 @@
+name: Docker Build HTML Image
+
+on:
+  # Allow manual trigger
+  workflow_dispatch:
+    inputs:
+      # Ask if docker depoyment needed when manually triggered
+      deploy:      
+        description: 'Deploy Docker Image (y/n)?'
```

Comment:
The workflow file has to be on master once to trigger the workflow even for testing even with a dispatch I guess, I will test it once it is on master if the pipeline build of the image works.

Reply:
@danwos I added a trigger to build on feature branch (temporarily) to trigger the build and it worked.
The image is building but the push to docker.io registry failed.
https://github.com/useblocks/sphinxcontrib-needs/runs/6263751379?check_suite_focus=true#step:7:1755
Could you please confirm that the Docker Token you added has the rights to push?

Reply:
@danwos Could you please let me know your dockerhub username? I assumed it is useblocks but cannot find the pushed image under this user on dockerhub :)

Reply:
Ohh I didn't know that there is a useblocks user.
I will try to reactivate it ...